Documentation/AppendixA/Index.rst

```diff
@@ -127,17 +127,17 @@ Internal variables in the main frontend object, TSFE
 
 There are some variables in the global object, TSFE (TypoScript
 Frontend), you might need to know about. These ARE ALL READ-ONLY!
-(Read: Don't change them!) See the class :php:`TypoScriptFrontendController`
-for the full descriptions.
+(Read: Don't change them!) See the class
+:php:`\TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController` for the
+full descriptions.
 
-.. important::
+You can retrieve the :php:`TypoScriptFrontendController` via the
+:ref:`request <typo3-request>` attribute
```

Comment:
These are found in TYPO3 Explained? The prefix would be needed for the link then.

Reply:
You are right. Seems like sphinx found them if they are unique in the intersphinx universe (so my preview was okay).